core/services/ardupilot_manager/rest_api.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+from typing import Any, Dict, List
+
+import uvicorn
+from ArduPilotManager import ArduPilotManager
+from fastapi import FastAPI
+from mavlink_proxy.Endpoint import Endpoint, EndpointType
+
+app = FastAPI()
+autopilot = ArduPilotManager()
+
+
+@app.get("/available_endpoints", response_model=List[Dict[str, Any]])
+def get_available_endpoints() -> Any:
```

Comment:
Can you explain why Any is used in return type and not List of Dict ?

Reply:
This is [a suggestion of the library's author](https://github.com/tiangolo/fastapi/issues/101#issuecomment-613242344), in fact.

On FastAPI, the types of the responses are defined by the response_model, not the typehint, for reasons he explains in the link before, and the function itself has parsers for serializing/deserializing, so their types not exactly represent the HTTP responses.

[There are libraries](https://fastapi-utils.davidmontague.xyz/user-guide/inferring-router/) that offer this funcionality, thought.